### PR TITLE
Remove elemental-cli from kairos

### PR DESCRIPTION
**Pull request by Itxaka**
Signed-off-by: Itxaka <itxaka.garcia@spectrocloud.com>

**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

### CODE REVIEW

**Comment by Itxaka** [comment]
Needs a repository bump to get the newer packages which include kairos-agent run stage and the new services using kairos-agent instead of elemental

**Comment by Itxaka** [comment]
Needs a repository bump to get the newer packages which include kairos-agent run stage and the new services using kairos-agent instead of elemental

File: scripts/build_vm.sh [hunk 1]
```diff
@@ -10,7 +10,7 @@ SSH_USER="${SSH_USER:-kairos}"
 SSHPASS="${SSH_PASS:-kairos}"
 export SSHPASS
 HD_SIZE="${HD_SIZE:-50000}"
-INSTALL_COMMAND="${INSTALL_COMMAND:-sudo /bin/sh -c 'elemental install /dev/sda && sync'}"
+INSTALL_COMMAND="${INSTALL_COMMAND:-sudo /bin/sh -c 'kairos-agent install /dev/sda && sync'}"
```
Comment (jimmykarily) [inline scripts/build_vm.sh:13]: Isn't it true that `kairos-agent install` and `kairos-agent manual-install` can be both used to achieve the same things (are interchangeable) ? If yes, maybe we should get rid of one of the 2.

Comment (jimmykarily) [reply scripts/build_vm.sh:13]: Nothing to do with this PR though, I just had that thought.

**Review by jimmykarily — DISMISSED** [review]

File: scripts/build_vm.sh (hunk 1 above)
Comment (Itxaka) [reply scripts/build_vm.sh:13]: Basically yes, but manual install accepts a config file while install does not and expects a config to be ready in the live CD or config dirs.

We should merge both and just add a flag to override the config file.

**Review by mauromorales — APPROVED** [review]

**Comment by Itxaka** [comment]
moved to https://github.com/kairos-io/kairos/pull/1441

**Pull request closed without merging**